keycloak_add_users.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/python
+import json
+import requests
+import sys
+import getopt
+
+class updateUserStruct:
+    def __init__(self, id, enabled, username, firstName, lastName, email, attributes):
+        self.id = id
+        self.enabled = enabled
+        self.username = username
+        self.firstName = firstName
+        self.lastName = lastName
+        self.email = email
+        self.attributes = attributes
+
+    def toJson(self):
+        return json.dumps(self, default=lambda o: o.__dict__)
+
+class createUserStruct:
+    def __init__(self, enabled, username, firstname, lastname, email, attributes, credentials):
+        self.enabled = enabled
+        self.username = username
+        self.firstName = firstname
+        self.lastName = lastname
+        self.email = email
+        self.attributes = attributes
+        self.credentials = credentials
+
+    def toJson(self):
+        return json.dumps(self, default=lambda o: o.__dict__)
+
+
+class userAttributes:
+    def __init__(self, first_name, last_name, account_id, account_number, org_id, is_internal, is_org_admin, is_active, entitlements):
+        self.first_name = first_name
+        self.last_name = last_name
+        self.account_id = account_id
+        self.account_number = account_number
+        self.org_id = org_id
+        self.is_internal = is_internal
+        self.is_org_admin = is_org_admin
+        self.is_active = is_active
+        self.entitlements = entitlements
+
+
+class userCredentials:
+    def __init__(self, temporary, type, value):
+        self.temporary = temporary
+        self.type = type
+        self.value = value
+
+    def toJson(self):
+        return [{'value': self.value, 'type': self.type, 'temporary': self.temporary}]
+
+
+class KeyCloakClient:
+    def __init__(self, baseUrl, username, password, accesstoken):
+        self.BaseURL = baseUrl
+        self.Username = username
+        self.Password = password
+        self.AccessToken = accesstoken
+
+    def startClient(self):
+        headers = {"Content-type": "application/x-www-form-urlencoded"}
+        response = self.rawMethod("POST", "/auth/realms/master/protocol/openid-connect/token",
```

Comment:
Would be nice to do some error handling here, what if the status code isn't what we expect?